bin/templates/scripts/cordova/lib/run.js

```diff
@@ -159,6 +158,7 @@ function checkDeviceConnected () {
  * @return {Promise}        Resolves when deploy succeeds otherwise rejects
  */
 function deployToDevice (appPath, target, extraArgs) {
+    events.emit('log', 'Deploying to device');
```

Comment:
It's it usually unclear where you're deploying to? If not, this should go to the verbose log IMHO

Reply:
This was meant more as a "structuring" comment between the "build" and the "deploy" phase, but as deploying to devices usually doesn't give any more output (although one might think about changing that) we could move this to verbose as well. Thoughts?

Reply:
I see. Sounds sensible. I can't play with iOS builds myself to get a feeling for it so I'll leave it to your judgement. Do as you see fit.